test/rules/file-header/bad/test.js.lint

```diff
@@ -4,9 +4,9 @@
  */
 
 export class A {
-    public x = 1;
```

Comment:
Since this is supposed to be a .js file test, I removed the visibility modifiers.